### PR TITLE
Multivariate Wilson Prior Blog Post

### PR DESCRIPTION
**Pull request by kmdalton**
Adding a blog post about the three multivariate Wilson prior pre-prints.

### CODE REVIEW

**Review by dennisbrookner — APPROVED** [review]

**Comment by dennisbrookner** [comment]
How does this work, am I meant to also merge this after approving?

**Comment by kmdalton** [comment]
My personal opinion is that it is best to wait for the PR's author to merge, but we should formalize this in a developer meeting.

**Comment by dennisbrookner** [comment]
Great, that was my plan too.

**Pull request merged**
